src/components/uku-chord/soundFiles.js

```diff
@@ -0,0 +1,23 @@
+import chordsDefinition from '../../chords'
+import { cleanNameForSounds } from '../../utils'
+
+const buildSoundFiles = () => {
```

Comment:
Since this is re-used for guitar and ukulele, you could move this up some levels and share it between both components

Reply:
Will do this as well